app/helpers/categories_helper.rb

```diff
@@ -6,7 +6,7 @@ def show_table(root, first, last)
           '</li>'+"\n"
     end
 
-    categories = (root.children.to_a).sort_by!{ |cat| cat.list_order }
+    categories = (root.children.to_a).sort_by!{ |cat| cat.list_order || cat.name }  ## sort by alpha if list_order is nil -dji
```

Comment:
I recognized this problem, but assumed that for any given universe, `list_order` would either be all filled in, or all null. Probably not a great assumption to make. Reverted.